internal/files/files.go

```diff
@@ -0,0 +1,90 @@
+package files
+
+import (
+	"bufio"
+	"encoding/json"
+	"io"
+	"os"
+
+	"github.com/Arti9991/shortener/internal/logger"
+	"github.com/Arti9991/shortener/internal/storage"
+	"go.uber.org/zap"
+)
+
+type FileStor struct {
+	ID       int    `json:"uuid"`
+	Shorturl string `json:"short_url"`
+	Origurl  string `json:"original_url"`
+}
+type FileData struct {
+	ID   int
+	stor *storage.Data
+	path string
+}
+
+func NewFiles(path string, stor *storage.Data) *FileData {
+	return &FileData{ID: 0, stor: stor, path: path}
+}
+
+// функция для чтения всех данных в файле и сохранения их в карту
+func (d *FileData) FileRead() {
```

Comment:
```suggestion
func (d *FileData) FileRead() error {
```
1. Функция должна транслировать наверх возникшие ошибки
2. Нет смысла делать операции с файлом, если мы, например, не смогли его открыть